srcpkgs/mesa/template

```diff
@@ -23,8 +23,8 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT, LGPL-2.1-or-later"
 homepage="https://www.mesa3d.org/"
 changelog="https://docs.mesa3d.org/relnotes.html"
-distfiles="https://mesa.freedesktop.org/archive/mesa-${version}.tar.xz"
-checksum=a2c584c8d57d3bd8ba11790a6e9ae3713f8821df96c059b78afb29dd975c9f45
+distfiles="https://archive.mesa3d.org/mesa-${version}.tar.xz"
+checksum=63236426b25a745ba6aa2d6daf8cd769d5ea01887b0745ab7124d2ef33a9020d
```

Comment:
why is the distfiles url changed?

Reply:
> why is the distfiles url changed?

They changed the url, if you use the base url, you'll be redirected on the following new site.